src/sentry/conf/server.py

```diff
@@ -1419,6 +1419,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 SENTRY_FEATURES: dict[str, bool | None] = {
     # Enables user registration.
     "auth:register": True,
+    # Enables region provisioning for individual users
+    "hybrid-cloud:region-provisioning-allow-list": False,
```

Comment:
The content before the `:` is a feature scope. It should be one of `user` `project` `system` or `organization`. Only `organization` scoped flags are sent to flagr currently.